src/transformers/models/code_llama/tokenization_code_llama.py

```diff
@@ -33,13 +33,17 @@
 PRETRAINED_VOCAB_FILES_MAP = {
     "vocab_file": {
         "hf-internal-testing/llama-code-tokenizer": "https://huggingface.co/hf-internal-testing/llama-tokenizer/resolve/main/tokenizer.model",
+        "codellama/CodeLlama-34b-Instruct-hf": "https://huggingface.co/codellama/CodeLlama-34b-Instruct-hf/resolve/main/tokenizer.model",
```

Comment:
remove these three lines and we can merge :)

(These are just here for backwards-compatibility)